### PR TITLE
Queue Handler for Flask app

### PR DESCRIPTION
**Pull request by jannisborn**
Solves #251

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -442,7 +437,64 @@ def change_machine():
...
+    request_queue.put((run_location_differ, ()))
+    return jsonify({"message": "Success!"}), 200
+
+
+def run_location_differ():
+    with setup_locdiffer_logger():
+        old_json_file = "/root/PennyMe/new_data/old_server_locations.json"
+        new_json_file = "/root/PennyMe/new_data/server_locations.json"
+        new_problems_json_file = "/root/PennyMe/new_data/problems.json"
+        debug_path = "/root/PennyMe/debug_new_data"
```
Comment (NinaWie) [inline backend/app.py:454]: Docstring missing, and shouldn't we set these paths as capital letter variables in the beginning of the file or so?

File: backend/app.py [hunk 2]
```diff
@@ -442,7 +437,64 @@ def change_machine():
...
+
+        # Make sure all preceding jobs are finished
+        wait()
+
+        location_differ(
+            output_folder="/root/PennyMe/new_data",
+            device_json="/root/PennyMe/data/all_locations.json",
+            server_json=old_json_file,
```
Comment (NinaWie) [inline backend/app.py:462]: same here

**Review by NinaWie — APPROVED** [review]
Great work on the queues!
Only minor comments

File: backend/app.py (hunk 1 above)
Comment (jannisborn) [reply backend/app.py:457]: Added the docstring, I'm against uppercasing because this is typically done for global not for local variables

**Pull request merged**